### PR TITLE
Allow multiple rows

### PR DESCRIPTION
**Pull request by diegosteiner**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
+  include Bitfields
```
Comment (sihu) [inline app/models/user.rb:4]: never used it before. looks cool 👍

File: app/models/esr_service.rb [hunk 2]
```diff
@@ -21,12 +21,12 @@ def format_ref(ref)
     ref.reverse.chars.in_groups_of(5).reverse.map { |group| group.reverse.join }.join(' ')
   end
 
-  def code_line(esr_participant_nr: '01-348230-2', ref:, amount:, esr_mode: '01')
+  def code_line(esr_participant_nr: '013482302', ref:, amount:, esr_mode: '01')
```
Comment (sihu) [inline app/models/esr_service.rb:24]: ok so it has to be a number?

File: app/models/user.rb [hunk 3]
```diff
@@ -9,7 +10,7 @@ class User < ApplicationRecord
   validates :uid, presence: true
   validates :pbs_id, presence: true, allow_blank: true
 
-  enum role: %i[user admin programm], _prefix: :role
+  bitfield :role_flags, :role_user, :role_admin, :role_programm
```
Comment (sihu) [inline app/models/user.rb:13]: this allows us multiple roles, right? is it easily adaptable if we would add more roles?

File: app/models/esr_service.rb (hunk 2 above)
Comment (diegosteiner) [reply app/models/esr_service.rb:24]: Nou, their arr leeding serows.

File: app/models/user.rb (hunk 3 above)
Comment (diegosteiner) [reply app/models/user.rb:13]: Just add more here, should work fine

**Pull request merged**